app/admin/order.rb

```diff
@@ -0,0 +1,12 @@
+ActiveAdmin.register Order do
+  index do
+    column :user
+    column :total
+    column "Line Items" do |order|
+      order.line_items.each do |line_item|
```

Comment:
Check out `table_for` on the following page:
http://activeadmin.info/docs/12-arbre-components.html

You can use a `table_for order.line_items` to make a subtable of line items.